syntax = "proto3";


message Empty {}

message Sender {
    string name = 1;
    string callsign = 2;
}

message AudioChunk {
    string id = 1;
    string sender = 2;
    bytes data = 3;
}

service AudioService {
    rpc SendAudio(AudioChunk) returns (Empty);
    rpc SendAudioStream(stream AudioChunk) returns (Empty);
    rpc GetAudioStream(Empty) returns (stream AudioChunk);
}